csm/cli/schema/s3accesskeys.json

```diff
@@ -140,20 +140,20 @@
     },
     {
       "name": "update",
-      "description": "Update the given access key",
+      "description": "Update the status of the given access key",
       "need_confirmation": true,
       "permissions_tag": "update",
       "args": [
         {
           "flag": "user_name",
           "type": "str",
-          "help": "S3 IAM user name for which access key needs to be updated.",
+          "help": "S3 IAM user name for which status of access key needs to be updated.",
           "params": true
         },
         {
           "flag": "access_key_id",
           "type": "str",
-          "help": "Access key id to be updated."
+          "help": "Access key id for which status need to be updated"
```

Comment:
need -> needs. Make sure that the line ending are consistent. Use full stop wherever required.

Reply:
sure @pawankumarsrivastava

Reply:
I have updated the changes @pawankumarsrivastava